app/views/questions/index.html.erb

```diff
@@ -1,4 +1,14 @@
 <div class="col-md-12">
+    <div class ="row">
+        <div class="col-md-6" >
+            <h4><b> <%= @stats if params[:period].present?  %></b></h4>
+        </div>
+        <div class="col-md-4">
+            <%= form_tag request.url, :method => 'get'  do %>
+                <%= select_tag :period, options_for_select(options),onchange: "this.form.submit();", include_blank: "Filter stats", class: "form-control"%>
```

Comment:
Ooh this is cool, could you share a screenshot? Thanks!

Reply:
@jywarren a screenshot is :point_up: above. Thanks for the feedback.